tests/tools.py

```diff
@@ -10,18 +10,16 @@
 import tempfile
 from contextlib import contextmanager
 from functools import wraps
+import pytest
 
-from nose.tools import assert_true, assert_equal
-from nose.plugins.attrib import attr
-from nose.plugins.skip import SkipTest
 
 from cyclus import lib as libcyclus
 
 if sys.version_info[0] >= 3:
     basestring = str
 
-unit = attr('unit')
-integration = attr('integration')
+#unit = attr('unit')
+#integration = attr('integration')
```

Comment:
If these are being commented out, can we remove them?

Reply:
I'm not sure these were every used in Cyclus... I think (!?!) these came over 10 years ago when this file was lifted from another project.